core/src/Ledger.sol

```diff
@@ -353,14 +345,17 @@ contract Ledger {
                 dt: block.timestamp - cache.lastAccrualTime
             });
 
-            cache.borrowIndex = (cache.borrowIndex * accrualFactor) / ONE;
-            cache.lastAccrualTime = 0; // 0 in storage means locked to reentrancy; 0 in `cache` means `borrowIndex` was updated
+            cache.borrowIndex = SoladyMath.min((cache.borrowIndex * accrualFactor) / ONE, type(uint72).max);
+            cache.lastAccrualTime = 0; // 0 indicates `borrowIndex` was updated
 
             uint256 newInventory = cache.lastBalance + (cache.borrowBase * cache.borrowIndex) / BORROWS_SCALER;
-            uint256 newTotalSupply = Math.mulDiv(
-                cache.totalSupply,
-                newInventory,
-                newInventory - (newInventory - oldInventory) / rf
+            uint256 newTotalSupply = SoladyMath.min(
+                SoladyMath.fullMulDiv(
+                    cache.totalSupply,
+                    newInventory,
+                    newInventory - (newInventory - oldInventory) / rf
+                ),
+                type(uint112).max
```

Comment:
We're clamping to the maximum value so that `accrueInterest` doesn't revert, guaranteeing that governance can always set a new rate model. Added in https://github.com/aloelabs/aloe-ii/pull/206